### PR TITLE
This PR includes changes to update docs and help text related to pdf test report generation

### PR DESCRIPTION
**Pull request by abhargav-mw**

### CODE REVIEW

File: CONFIGDOC.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Select check boxes in the **Generate Test Artifacts** and **Generate Coverage Ar
 The **Run MATLAB Tests** build step produces a MATLAB script file and uses it to run the tests and generate the artifacts. The plugin writes the contents of this file to the build log. You can review the build log in **Console Output** to understand the testing workflow.
  
 Artifacts to generate with the plugin are subject to these restrictions: 
-* Producing a PDF test report is not currently supported on macOS platforms.
+* Producing a PDF test report is supported on macOS platforms in MATLAB R2020b and later.
```
Comment (mw-hrastega) [inline CONFIGDOC.md:88]: Producing a PDF test report on macOS platforms is supported in MATLAB R2020b and later.

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html [hunk 2]
```diff
@@ -7,5 +7,5 @@
 Consider copying the report to a location outside the workspace and opening it from there. For more information, 
 see <a href="http://wiki.jenkins.io/display/JENKINS/Configuring+Content+Security+Policy">Configuring Jenkins Content Security Policy</a>.
 <br><br>
-<b>Note:</b> This feature is not currently supported on Mac platforms.
+<b>Note:</b> This feature is supported on Mac platforms in MATLAB R2020b and later.
```
Comment (mw-hrastega) [inline src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html:10]: On macOS platforms, this feature is supported in MATLAB R2020b and later.

File: CONFIGDOC.md (hunk 1 above)
Comment (abhargav-mw) [reply CONFIGDOC.md]: Done

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html (hunk 2 above)
Comment (abhargav-mw) [reply src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html]: Done

**Comment by abhargav-mw** [comment]
@mw-hrastega We will need to update docs on other CI services as well.

File: CONFIGDOC.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ Select check boxes in the **Generate Test Artifacts** and **Generate Coverage Ar
 The **Run MATLAB Tests** build step produces a MATLAB script file and uses it to run the tests and generate the artifacts. The plugin writes the contents of this file to the build log. You can review the build log in **Console Output** to understand the testing workflow.
  
 Artifacts to generate with the plugin are subject to these restrictions: 
-* Producing a PDF test report is not currently supported on macOS platforms.
+* Producing a PDF test report on macOS platforms is supported in MATLAB R2020b and later.
```
Comment (mw-hrastega) [inline CONFIGDOC.md:88]: This is good, but in the same file we have a table for pipeline syntax. I still see the outdated text in it:

(Optional) Path to write test results report in PDF format. Currently, this input is not supported on macOS platforms.
Example: 'test-results/results.pdf'

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
I think we need to update the text in one more place.

File: CONFIGDOC.md [hunk 4]
```diff
@@ -273,7 +273,7 @@ Input                     | Description
...
 `selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To create a test suite, the plugin uses only the tests in the specified folder and its subfolders.<br/>**Example:** `['test']`<br/>**Example:** `['test/folderA', 'test/folderB']`
 `selectByTag`             | (Optional) Test tag used to select test suite elements. To create a test suite, the plugin uses only the test elements with the specified tag.<br/>**Example:** `'FeatureA'`
-`testResultsPDF`          | (Optional) Path to write test results report in PDF format. Currently, this input is not supported on macOS platforms.<br/>**Example:** `'test-results/results.pdf'`      
+`testResultsPDF`          | (Optional) Path to write test results report in PDF format.<br/>**Example:** `'test-results/results.pdf'`
```
Comment (mw-hrastega) [inline CONFIGDOC.md:276]: Can't we say this?

On macOS platforms, this input is supported in MATLAB R2020b and later.

**Review by mw-hrastega — CHANGES_REQUESTED** [review]

File: CONFIGDOC.md (hunk 3 above)
Comment (abhargav-mw) [reply CONFIGDOC.md:88]: Done.

File: CONFIGDOC.md (hunk 4 above)
Comment (abhargav-mw) [reply CONFIGDOC.md]: Done.

**Review by mw-hrastega — APPROVED** [review]

**Pull request merged**